flink-staging/flink-tez/src/main/java/org/apache/flink/tez/runtime/TezTask.java

```diff
@@ -402,7 +402,7 @@ private void initInputLocalStrategy(int inputNum) throws Exception {
 						localStub = initStub(userCodeFunctionType);
 					} catch (Exception e) {
 						throw new RuntimeException("Initializing the user code and the configuration failed" +
-								e.getMessage() == null ? "." : ": " + e.getMessage(), e);
+								(e.getMessage() == null ? "." : ": ") + e.getMessage(), e);
```

Comment:
It seems wrong parenthesis. If `e.getMessage()` is `null`, there is concatenation of `"."` and `null`.